models/DataObject/Classificationstore.php

```diff
@@ -467,6 +475,10 @@ public static function doGetFallbackValues()
      */
     public function getGroupCollectionMappings(): array
     {
+        $doGetInheritedValues = Model\DataObject::doGetInheritedValues();
```

Comment:
using DataObject:doGetInheritedValues(); to avoid breaking Changes

 has it is done in getLocalizedKeyValue()